src/astro/databases/base.py

```diff
@@ -201,15 +201,44 @@ def append_table(
         self,
         source_table: Table,
         target_table: Table,
-        if_conflicts: AppendConflictStrategy = "exception",
+        source_tables_cols: Optional[List[str]] = None,
+        target_tables_cols: Optional[List[str]] = None,
     ):
         """
-        Append the source table rows into a destination table.
-        The argument `if_conflicts` allows the user to define how to handle conflicts.
+        Append the source table rows into a destination table
+        The argument `conflict_strategy` allows the user to define how to handle conflicts
 
         :param source_table: Contains the rows to be appended to the target_table
         :param target_table: Contains the destination table in which the rows will be appended
-        :param if_conflicts: The strategy to be applied if there are conflicts.
+        :param conflict_strategy: Action that needs to be taken in case there is a conflict
+        :param source_tables_cols: List of columns name in source table that will be used in appending
+        :param target_tables_cols: List of columns name in target table that will be used in appending
+        """
+        return self.merge_table(
+            source_table=source_table,
+            target_table=target_table,
+            source_tables_cols=source_tables_cols,
+            target_tables_cols=target_tables_cols,
+        )
+
+    def merge_table(
```

Comment:
The end goal was to have a single method for append and merge, with a single `Strategy` (previously defined). why have we created a new method?

Reply:
@tatiana merge and append can still be wildly different in implementation even if it looks like one "endpoint" to the user. This is why I have not been a fan of merging these functions.

Reply:
We have started to use the new function combine_tables() function for all purposes.